loops_lab.js

```diff
@@ -0,0 +1,111 @@
+//1. Write a `while` loop and a `for` loop that takes the variable
```

Comment:
Good demonstration of how to use for and while loops, and implementation. Try to do bonus problems next time. Those are always good practice for you. Keep up the good work!